packages/protocol/contracts/common/Accounts.sol

```diff
@@ -0,0 +1,464 @@
+pragma solidity ^0.5.3;
+
+import "openzeppelin-solidity/contracts/utils/ReentrancyGuard.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+import "./interfaces/IAccounts.sol";
+
+import "../common/Initializable.sol";
+import "../common/Signatures.sol";
+import "../common/UsingRegistry.sol";
+
+contract Accounts is IAccounts, ReentrancyGuard, Initializable, UsingRegistry {
+
+  using SafeMath for uint256;
+
+  struct Signers {
+    // The address that is authorized to sign a vote on behalf of the account.
+    // The account can vote as well, whether or not an vote signing key has been specified.
+    address voting;
+
+    // The address that is authorized to sign consensus messages on behalf of the account.
```

Comment:
`The address that is authorized to manage a validator or validator group and sign consensus messages on behalf of the account.`